lib/utils/authentication.js

```diff
@@ -9,20 +9,20 @@ const Mcrypt = require('mcrypt').MCrypt;
 const cypher = new Mcrypt('rijndael-128', 'ecb');
```

Comment:
I've had add mcrypt to cookie-utils to encrypt a username for a db call, should we have a general purpose en/decrypt funcs in here?